column/__init__.py

```diff
@@ -9,7 +10,7 @@
 
 import ConfigParser
 
-DEFAULT_CONF_FILE = '/etc/column/column.conf'
+DEFAULT_CONF_FILE = os.path.join('/etc/column', 'column.conf')
```

Comment:
Also you should have / at all in the strings.  Use:

os.path.join('etc', 'column', 'column.conf')